### PR TITLE
fix: var to let or const（#610）

### PR DESCRIPTION
**Pull request by xiaodaiweiwei**
Fixes #610 

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [✅ ] PR is descriptively titled 📑 and links the original issue above 🔗
* [ ✅] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `grunt test`
* [ ✅] code is in uniquely-named feature branch and has no merge conflicts 📁
* [ ] screenshots/GIFs are attached 📎 in case of UI updates
* [✅] @mention the original creator of the issue in a comment below for help or for a review

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

**Comment by xiaodaiweiwei** [comment]
@TildaDares I would appreciate a review of my PR. Thanks!

**Review by TildaDares — CHANGES_REQUESTED** [review]
Hi @xiaodaiweiwei, it seems your IDE added additional files. Can you please remove them? Thanks!

File: src/util/IconUtil.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ L.IconUtil = {
   },
 
   addClassToSvg(container, loader) {
-    var svg = container.querySelector('svg');
+    let svg = container.querySelector('svg');
```
Comment (TildaDares) [inline src/util/IconUtil.js:20]:
```suggestion
    const svg = container.querySelector('svg');
```

**Review by TildaDares — CHANGES_REQUESTED** [review]
Hi @xiaodaiweiwei, let's use `const` instead since the variables aren't changing. Thanks!

File: src/util/IconUtil.js (hunk 1 above)
Comment (xiaodaiweiwei) [reply src/util/IconUtil.js:20]: Ok! I would change that!

**Comment by xiaodaiweiwei** [comment]
I have made the changes. Please check them. Thank you @TildaDares

**Comment by TildaDares** [comment]
Hi @xiaodaiweiwei, you can try closing this PR and opening a PR to remove those files. Thanks!

**Pull request closed without merging**